docs/docs/references/applications/README.mdx

```diff
@@ -0,0 +1,30 @@
+# 📱 Applications
+
+## Introduction
+
+Each real-world application that requires user authentication using Logto needs to be registered as an application in Logto.
```

Comment:
nit: sounds a little strange. Can we make it clear?

Reply:
Yes this sounds unnatural

Reply:
Yes this sounds unnatural